lms/envs/common.py

```diff
@@ -1903,6 +1903,12 @@
     'ROOT_PATH': '/tmp/edx-s3/grades',
 }
 
+DETAILED_ENROLLMENT_REPORT = {
+    'STORAGE_TYPE': 'localfs',
+    'BUCKET': 'edx-grades',
+    'ROOT_PATH': '/tmp/edx-s3/grades',
+}
+
```

Comment:
I'd suggest making this a bit more generic, I think we'll be putting all financial reports in there, so how about "FINANCIAL_REPORTS"?

Note, we'll have to add some code to aws.py to be able to read this configuration out of the lms.env.json file. Just look for what "GRADES_DOWNLOAD" does in aws.py and do the same for this configuration variable.